ccvm_simulators/solvers/dl_solver.py

```diff
@@ -228,6 +276,83 @@ def tune(self, instances, post_processor=None, pump_rate_flag=True, g=0.05):
         #       future consideration
         self.is_tuned = True
 
+    def optics_machine_energy(self, machine_parameters=None):
+        """The wrapper function of calculating the maximum energy consumption of the
+        solver simulating on a DL-CCVM machine.
+
+        Args:
+            machine_parameters (dict, optional): Parameters of the. Defaults to None.
```

Comment:
I think you're missing a few words here!

Reply:
Thanks for catching that. I've fixed it.